packages/portalnetwork/src/subprotocols/beaconChain/types.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  ByteListType,
+  ByteVectorType,
+  ContainerType,
+  ListCompositeType,
+  UintBigintType,
+  UnionType,
+  BitVectorType,
+} from '@chainsafe/ssz'
+
+// todo: export
+
+/* ----------------- Alliases ----------------- */
+const Bytes4 = new ByteVectorType(4)
+const Bytes32 = new ByteVectorType(32)
+const Bytes48 = new ByteVectorType(48)
+const Bytes96 = new ByteVectorType(96)
+/* ----------------- Types ----------------- */
+
+const Slot = new UintBigintType(64)
+const Epoch = new UintBigintType(64)
+const CommitteeIndex = new UintBigintType(64)
+const ValidatorIndex = new UintBigintType(64)
+const Gwei = new UintBigintType(64)
+const GeneralizedIndex = new UintBigintType(64)
```

Comment:
For Uint types, you'll want to pass in the `bytelength` instead.
So for a "Uint64" type, like Gwei, you would  do:
`const Gwei = new UintBigIntType(8)`

On the SSZ level, there is no difference between `UintNumberType(X)` and `UintBigIntType(X)`, but JS numbers can only reach `MAX_SAFE_INTEGER = 2^53 - 1`, so for Uint64 and larger it is generally safer to use `UintBigIntType`.